client/src/components/Icon.jsx

```diff
@@ -0,0 +1,21 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+function Icon({ id, size = 24 }) {
```

Comment:
Since we're using `prop-types` now, I think we don't have to set the default up here. Try removing the `= 24` part and seeing if it still works since `Icon.defaultProps` for `size` is listed below.